.github/workflows/code_format.yml

```diff
@@ -17,8 +17,8 @@ jobs:
           repository: ${{ github.event.pull_request.head.repo.full_name }}
       - uses: DoozyX/clang-format-lint-action@v0.12
         with:
-          source: '. ./src'
-          exclude: './src/libs ./test ./scripts'
+          source: '. ./src ./src/libs'
+          exclude: './src/libs/TimerInterrupt ./test ./scripts'
```

Comment:
Since I changed this file the GitHub workflow will use the unchanged version for this PR but future PRs will have the change incorporated. (this is due to their PR security model)